src/components/AutoComplete/AutoComplete.component.tsx

```
@@ -0,0 +1,57 @@
+                            if (e.key === 'Enter') {
+                                handleSelection(e.currentTarget.value);
+                            }
+                        },
+                        startAdornment: (
+                            <InputAdornment position="start">
+                                {StartIcon && <StartIcon />}
+                            </InputAdornment>
+                        ),
+                        endAdornment: (
+                            <InputAdornment position="end">
+                                <>{EndIcon || params.InputProps.endAdornment}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)